katas/README.md

```diff
@@ -38,6 +43,12 @@ The following macros are available for katas composition:
     - id: Unique identifier for the section.
     - title: Title of the section.
 
+### Build Katas Locally
```

Comment:
I would move this info up, just below "Run Katas Online" and make it same lever header as that one, since you have to build the katas locally to make use of the rest of the information about the macros and such.

I'd also direct the reader to follow the building instructions in the playground readme (with a link) rather than repeat them here. If we update the build process for the playground, there's no way we'll remember to update them here as well, so it's better to have instructions in one place and links to them from all others.

Reply:
Done. Actually, I thought along those lines initially and then thought it might be an additional redirection for a short command. 

Since, the main README is copy-paste of instructions to run playground, I did the similar change there as well.